benchmarks/timeseries/resampling.py

```diff
@@ -0,0 +1,51 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Benchmark resampling."""
+
+from datetime import datetime, timedelta, timezone
+from timeit import timeit
+from typing import Sequence
+
+from frequenz.sdk.timeseries import Sample
+from frequenz.sdk.timeseries._resampling import ResamplerConfig, _ResamplingHelper
+
+
+def nop(  # pylint: disable=unused-argument
+    samples: Sequence[Sample], resampling_period_s: float
+) -> float:
+    """Return 0.0."""
+    return 0.0
+
+
+def _benchmark_resampling_helper(resamples: int, samples: int) -> None:
```

Comment:
Can we start writing in the benchmark files information when we last tested it and what was the result?
We could have short history with date and result.

Reply:
That's a good idea, but I'm not sure I would write it in the benchmark file itself, maybe a Wiki? Also, if we do this, we need to take it with a **BIG** grain of salt, because results will be completely different when ran in different hardware, so at the end I'm not sure this will provide any useful information, unless we have some sort of standard hardware to run the benchmarks on.

Reply:
hmmm true... maybe `microgrid.sandbox.api.frequenz.io`.
I would write it in file. Otherwise everyone would forget it, and it will be harder to track history of benchmark files.

Reply:
I created an internal issue to keep track of this to avoid blocking this PR with a topic that is much broader.